developer-docs/bolt-api-servers.md

```diff
@@ -380,8 +380,26 @@ This endpoint behaves identically to the /ssh/check_node_connections endpoint, b
 
 This endpoint parses a project inventory and returns a list of target hashes. Note that the only accepetable inventoryfile location is the default `inventory.yaml` at the root of the project.
 
+### Query parameters
```

Comment:
This is a nitpick, but these new sections looks like they should be subsections of the `POST /project_inventory_targets` section, so maybe these headers should be the level below the proj_inv_targets header so they don't all look like separate sections?